cluster/network.go

```diff
@@ -454,9 +524,19 @@ func (c *Cluster) runServicePortChecks(ctx context.Context) error {
 	return errgrp.Wait()
 }
 
-func checkPlaneTCPPortsFromHost(ctx context.Context, host *hosts.Host, portList []string, planeHosts []*hosts.Host, image string, prsMap map[string]v3.PrivateRegistry) error {
+func checkPlanePortsFromHost(ctx context.Context, host *hosts.Host, portList []string, planeHosts []*hosts.Host, image string, prsMap map[string]v3.PrivateRegistry, tcp bool) error {
 	var hosts []string
 
+	cmd := []string{
+		"sh",
+		"-c",
+	}
+	if tcp {
+		cmd = append(cmd, "for host in $HOSTS; do for port in $PORTS ; do echo \"Checking host ${host} on port ${port}\" >&1 & nc -w5 -z $host $port > /dev/null || echo \"${host}:${port}\" >&2 & done; wait; done")
+	} else {
+		cmd = append(cmd, "for host in $HOSTS; do for port in $PORTS ; do echo \"Checking host ${host} on port ${port}\" >&1 & nc -w5 -uzv $host $port > /dev/null & done; wait; done")
```

Comment:
Why did you omit the fail check `|| echo \"${host}:${port}\"` ?

Reply:
Because `UDP` port scans using the `-uz` combination of flags will always report success irrespective of the target machine's state, so instead we use `-uzv` to log verbose output if the checking port is not open or unreachable, and it will print nothing if it succeeds.

Reply:
@guangbochen Can you please add that as a comment ?

Reply:
addressed @moelsayed.